metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -373,7 +374,8 @@ case class DebugUnresolvedMainClassParams(
     mainClass: String,
     @Nullable buildTarget: String = null,
     @Nullable args: java.util.List[String] = null,
-    @Nullable jvmOptions: java.util.List[String] = null
+    @Nullable jvmOptions: java.util.List[String] = null,
+    @Nullable env: java.util.List[String] = null
```

Comment:
Can we also update the docs here: https://scalameta.org/metals/docs/editors/new-editor.html#start-debug-adapter to show this in the example?

Reply:
Good thinking! I think I will also change it to Map[String, String] if possible

Reply:
Fixed!